pkg/cloud/services/networking/securitygroups.go

```diff
@@ -200,7 +202,29 @@ func (s *Service) generateDesiredSecGroups(openStackCluster *infrav1.OpenStackCl
 	workerRules := append([]resolvedSecurityGroupRuleSpec{}, defaultRules...)
 
 	controlPlaneRules = append(controlPlaneRules, getSGControlPlaneHTTPS()...)
-	workerRules = append(workerRules, getSGWorkerNodePort()...)
+
+	// Fetch subnet to use for worker node port rules
+	// In the future IPv6 support need to be added here
```

Comment:
IPv4-only is icky, but everything else is still IPv4 only and it's not worth fixing it piecemeal so this is fine.